.github/workflows/schematics.yml

```diff
@@ -0,0 +1,49 @@
+name: ⚙️ Schematics
+on:
+  pull_request:
+    paths:
+      - 'projects/cdk/schematics/**'
+
+jobs:
+  build-cdk:
+    if: ${{ !contains(github.head_ref , 'release/') }}
+    name: Build @taiga-ui/cdk
+    runs-on: ubuntu-latest
+    steps:
+      - uses: taiga-family/ci/actions/setup/checkout@v1.51.14
+      - uses: taiga-family/ci/actions/setup/variables@v1.51.14
+      - uses: taiga-family/ci/actions/setup/node@v1.51.14
+
+      - run: npx nx build cdk
+
+      - name: Upload cache / ${{ env.CACHE_DIST_KEY }}
+        uses: actions/cache/save@v4.0.1
+        with:
+          path: dist/cdk
+          key: ${{ env.CACHE_DIST_KEY }}
```

Comment:
`key: ${{ env.CACHE_DIST_KEY }}_cdk`

Reply:
Fixed in 5d3df851a88dcda8ed3a9d378acc16d280c654ae